SpinCore_pp/ppg/echo.py

```diff
@@ -0,0 +1,78 @@
+from .. import configureRX, configureRX, init_ppg stop_ppg, runBoard, getData
+from .. import load as spincore_load
+def run_spin_echo(nScans, indirect_idx, indirect_len, DNP_data=None):
+    """run nScans and slot them into the indirect_idx index of DNP_data -- assume
+    that the first time this is run, it will be run with DNP_data=None and that
+    after that, you will pass in DNP_data
+    
+    this generates an "indirect" axis
+    """
+    print("about to run run_spin_echo for",indirect_idx)
+    ph1_cyc = r_[0,1,2,3]
+    ph2_cyc = r_[0]
```

Comment:
really, this should be a keyword argument instead, because you are setting it at the top of your example script

Reply:
made into kwargs